src/main/java/com/azure/cosmos/kafka/connect/sink/CosmosDBSinkConfig.java

```diff
@@ -15,54 +15,11 @@
 
 @SuppressWarnings ({"squid:S1854", "squid:S2160"})  // suppress unneeded int *groupOrder variables, equals method
 public class CosmosDBSinkConfig extends CosmosDBConfig {
-
-    static final String COSMOS_USE_UPSERT_CONF = "connect.cosmos.sink.useUpsert";
-    private static final String COSMOS_USE_UPSERT_DEFAULT = "false";
```

Comment:
these changes are part of another PR, right?

Reply:
I ended up stacking this on #382 as the separation was not entirely clean. Part of the motivation for removing the schema-registry dependency was that it was pulling in Jersey, which is both unnecessary and contained a CVE.

Reply:
does that make the other PR redundant now?